src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -183,6 +183,30 @@ public void getInvalidPathInfoTest() throws IOException, ServletException {
         assertEquals(expected, actual);
     }
 
+    // Get a similar meal from Datastore
+    // Expected result: String in JSON format with id of second meal.
```

Comment:
Expected result is a number / a Long returned in JSON format, i.e. `2`, not a string in JSON format (`"2"`). Please change this comment accordingly.